### ISSUE TITLE
Template variables within custom header

### ISSUE THREAD

**Issue by lmaylein** [IQSS/dataverse#9208]
Some time ago I suggested to extend the template dataverse_header.xhtml so that different dataverses can be treated differently by CSS, for example.

`<div id="dataverse-header-block" data-alias="#{dataverse.alias}">`  

instead of

`<div id="dataverse-header-block">  `

In the meantime, however, there is the nice feature for a custom header and a custom stylesheet. I would have liked something like this as custom header:

`<div class="my_header" data-alias="#{dataverse.alias}">`  

Unfortunately, however, the template variables are not evaluated for these custom templates. Can this be changed easily, so that the custom stylesheets are even more flexible usable (for example different max-width for dataverse logos etc.)?

**Comment by pdurbin** [comment]
@lmaylein yes, now that I'm looking at #4917 and the [discussion](https://groups.google.com/g/dataverse-community/c/YxjFYM9GWuo/m/pZlq6LflGAAJ) on the mailing list @philippconzett started, I'm reminded that there's a desire for more configurability of design/css at the collection (dataverse) level.

As you may be aware, we're thinking about rewriting the UI in a different technology as [discussed](https://groups.google.com/g/dataverse-community/c/akcFYv3fcGo/m/xk0VQiY1BAAJ) on the mailing list. My preference would be to wait for the dust to settle there before considering this issue.

**Comment by cmbz** [comment]
2025/01/27: Closing as unplanned because of SPA work as well as work happening to make collections more configurable.
